src/fee_simulator/core/refunds.py

```
@@ -3,66 +3,71 @@
+      money out = every `earned` credit (validators, leaders, appellant
+                  rewards, explicit sender legs) + residual appellant bond
+                  burns (paths where the simulator still destroys a bond)
+                  + this refund
 
-        if event.role == "APPEALANT":
-            if event.earned > 0 and event.round_index is not None:
-                # Find the most recent normal round before this appeal
-                normal_round_index = find_previous_normal_round(event.round_index, round_labels)
-                if normal_round_index is None:
-                    normal_round_index = event.round_index - 1  # Default fallback
+    ⇒ refund = total_cost + Σ bonds − Σ earned − Σ appellant burns
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)